exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -189,6 +189,15 @@ drill.exec: {
   debug: {
     return_error_for_failure_in_cancelled_fragments: false
   }
+  udf: {
+    retry.times: 5,
+    directory: {
+      base: "/tmp",
+      staging: ${drill.exec.zk.root}"/udf/staging",
```

Comment:
Got it. Agree that paths must include the zk.root to keep them distinct. (Strangely, different cluster IDs overlap in other ZK settings, so you are correct to not include the cluster id.)